.github/workflows/doc-publish.yml

```diff
@@ -63,17 +63,8 @@ jobs:
           NETLIFY_MESSAGE: ${{ steps.source-run-info.outputs.pullRequestNumber }}
           NETLIFY_ALIAS: deploy-preview-${{ steps.source-run-info.outputs.pullRequestNumber }}
 
-      # Add deployment as status check, PR comment and annotation
+      # Add deployment as status check and annotation
       # we could use the nwtgck/actions-netlify action for that, except for that it is not (yet) working in workflow_run context: https://github.com/nwtgck/actions-netlify/issues/545
-      - name: Add/Update deployment status PR comment
-        uses: marocchino/sticky-pull-request-comment@v2
-        with:
-          number: ${{ steps.source-run-info.outputs.pullRequestNumber }}
-          header: preview-comment
-          recreate: true
```

Comment:
So, if we just drop this line. Then the noise is gone as well and we get a single comment that updates (silently) instead?

Reply:
Would that be an acceptable approach to @kwankyu and @mkoeppe  (and everybody else of course)?

Reply:
No. Why do you ask the same question again? It has been clearly answered. The notification is what people who are doing the work want to get.

Reply:
@mkoeppe , actually no. The notification gets on my nerves. But more importantly, it may lead to me not noticing important messages, because I click "mark all as read"

Reply:
> Why do you ask the same question again?

I do by no means see this as a repeated question.

And I am trying to find some middle ground here between those who are annoyed by these notifications and those who want to keep them. With the proposed change, you still get all the relevant links. But if you want push notifications, you'd have to get these by other means.

> The notification is what people who are doing the work want to get.

I hope you are not trying to imply anything with "people who are doing the work". There seems to be a substantial amount of people here who do **not** want these notifications.

So, unless I misunderstood you here. You want a notification and email every time the workflow finishes. So you want, what others consider spam. That's your opinion and you are entitled to it. But since that's precisely what others do not want, we have apparently no hope to come to a compromise here.